### PR TITLE
fix alpaca regular expression mismatch issue

### PR DESCRIPTION
**Pull request by twangxxx**

### CODE REVIEW

File: R/panda.to.condor.object.R [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #' @return a CONDOR object, see \code{\link{create.condor.object}}.
 #' @import viridisLite
 #' @examples 
+#' \dontrun{
```
Comment (twangxxx) [inline R/panda.to.condor.object.R:17]: @marouenbg , I've been working on fixing this issue these days, but it doesn't work.

In [build 120](http://zookeeper.networkmedicine.org/job/netZooR_Ubuntu18.04/lastBuild/console), it's saying the build failed on checking example of the function `panda.to.condor.object,` which is where I pointed out.
Theoretically, it won't be tested in R CMD check as I add `\dontrun ` here. However, in jenkins, it ran this example and failed in this example code.

Also, [build 120](http://zookeeper.networkmedicine.org/job/netZooR_Ubuntu18.04/lastBuild/console), it tested three versions of netZooR pkg, 0.7.1, 0.7.2(idk why this happened), and 0.7.3. **In this PR, the based branch devel is netZooR 0.7.2 and in this PR is still 0.7.2.** It doesn't make sense the build faild in version 0.7.3. 

I once made a uptick to v0.7.3 in #156, but that PR was deprecated and never merge into any branch. namely, the netzoor current version is 0.7.2 in github. 

I guess there is somewhat legacy of previous jenkins builds history, which is disrupting the current build.

Any thought on this issue?

**Comment by marouenbg** [comment]
running double-threaded now. Last duration 8h6mn

**Comment by twangxxx** [comment]
hello @marouenbg , (sorry to bother you on Saturday), this most recent build still took 8 hrs to run, maybe the double-threaded didn't work well as we expected. Since I have other bugs in source code to fix, may I merge this PR and continue my work first?

**Comment by marouenbg** [comment]
Hi @twangxxx , Jenkins ran single-threaded again so I need to check how to enable parallel processing. Yes go ahead.

**Pull request merged**